tools/who_what_benchmark/whowhatbench/visualtext_evaluator.py

```diff
@@ -0,0 +1,175 @@
+from typing import Any, Union
+
+import datasets
+import pandas as pd
+import requests
+from optimum.intel.openvino.modeling_visual_language import \
+    MODEL_TYPE_TO_CLS_MAPPING
+from PIL import Image
+from tqdm import tqdm
+from transformers import set_seed
+
+from .registry import register_evaluator
+from .text_evaluator import TextEvaluator
+
+
+def get_pil_from_url(url):
```

Comment:
maybe useful to extend this functional to load local images too like https://github.com/huggingface/diffusers/blob/main/src/diffusers/utils/loading_utils.py#L13 in case if dataset may be changed in future and for optimizing loading time for vaidation

Reply:
what if I just import this function from there?

Reply:
yes, I think it is possible, the only thing it will requires to have diffusers at the moment of VLM models evaluation only for images loading. I found similar helper in transformers too, so probably to use it https://github.com/huggingface/transformers/blob/eed11f34abe558c76590366a21316318e5f820ed/src/transformers/image_utils.py#L344

Reply:
replaced